validation/js/src/main/scala/hmda/validation/rules/ts/syntactical/S028.scala

```diff
@@ -0,0 +1,15 @@
+package hmda.validation.rules.ts.syntactical
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.dsl.{ Result, HmdaDsl, CommonDsl }
```

Comment:
I haven't found a way to share these implementations, which while identical call different implementations of the date formatting function, given that we're targeting two runtimes (JVM and JS). Now, this project would be a lot simpler to maintain if we forget about JS. If we keep it there is additional complexity and yes, and additional maintenance burden.